### DISCUSSION TITLE
Throwable Player

### DISCUSSION THREAD

**Discussion by Zoey2936** [SanCraftDev/Stacker#6] [Ideas]
Make stacked players thrown from the stacker and that they can jump off the stacker itself, preferably with an additional option in the configuration, a permission and a command. That would be perfect!

**Comment by DavidCraftDev** [comment]
Already in the works and will be released in the near future

**Reply by Zoey2936** [reply]
"Near Futur", next year?

**Reply by DavidCraftDev** [reply]
hehe

**Reply by DavidCraftDev** [reply]
I hope someone does this themselves and makes a pr 🌚

**Reply by DavidCraftDev** [reply]
> "Near Futur", next year?

I would say never xD

**Reply by DavidCraftDev** [reply]
Cloud come this year xD
